docs/en/index.rst

```diff
@@ -21,11 +21,11 @@ Load the plugin by adding the following statement in your project's ``src/Applic
 Getting Started
 ===============
 
-Add the authentication to the middleware. See the CakePHP `documentation
-<http://book.cakephp.org/3.0/en/controllers/middleware.html>`_ on how to use
-middleware if you are not familiar with it.
-
-Example of configuring the authentication middleware using ``authentication`` application hook::
+The authentication plugin integrates with your application as a middleware
+`middleware <http://book.cakephp.org/4/en/controllers/middleware.html>`_, and
+optionally as a component to make allowing unauthenticated access simpler. First, lets
+apply the middleware. In **src/Application.php** add the following to the class
+imports::
```

Comment:
```suggestion
The authentication plugin usually integrates with your application as a middleware
`middleware <http://book.cakephp.org/4/en/controllers/middleware.html>`_. It can also
be used as a component to make unauthenticated access simpler. First, lets
apply the middleware. In **src/Application.php**, add the following to the class
imports::
```

Reply:
@othercorey That kinda makes it sound as if the middleware was optional.

Reply:
Removed the word 'usually'.